### ISSUE TITLE
Process names (e.g. explorer.exe) instead of window titles by aw-watcher-window

### ISSUE THREAD

**Issue by Fivefold** [ActivityWatch/activitywatch#674]
- [x] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version. 
 - [x] I have searched the issues of this repo and believe that this is not a duplicate.

- **OS name and version**: Windows 10 Pro 20H2 build Edition	Windows 10 Pro build 19042.1237
- **ActivityWatch version**: 0.11.0

## Describe the bug
Not sure if this is actually a bug or intended behaviour, but the official screenshots imply otherwise. For window titles `aw-watcher-window` seems to log the process names (e.g. `firefox.exe` instead of `Firefox`). This also affects category matching, e.g. (case-sensitive) `Firefox` won't match the firefox application, but `firefox.exe` or (case-sensitive) `firefox` will.

This is on a fresh install.

Maybe it's relevant, I installed ActivityWatch using chocolatey https://community.chocolatey.org/packages/activitywatch

## Expected behavior
Show the window titles as shown in the official screenshots.

<img width="245" alt="activitywatch3" src="https://user-images.githubusercontent.com/38764218/136532280-263853fb-c62c-4b16-b395-3fb9656d4df2.png">

## Documentation

<img width="272" alt="activitywatch1" src="https://user-images.githubusercontent.com/38764218/136532332-8b4c5390-060f-4e3d-9542-8f5bff0dcb99.png">
<img width="566" alt="activitywatch2" src="https://user-images.githubusercontent.com/38764218/136532341-0d29b8ef-1288-4bf3-a6c2-ab91a58c892c.png">

## Additional context
I found errors in the aw-watcher-window logs but don't think they are related:

```
2021-10-07 20:18:55 [INFO ]: aw-watcher-window started  (aw_watcher_window.main:51)
2021-10-07 20:18:56 [INFO ]: Connection to aw-server established by aw-watcher-window  (aw_client.client:419)
2021-10-07 20:53:53 [ERROR]: Exception thrown while trying to get active window: (-2147217385, 'OLE error 0x80041017', None, None)  (aw_watcher_window.main:75)
2021-10-07 20:55:52 [ERROR]: Exception thrown while trying to get active window: (-2147217385, 'OLE error 0x80041017', None, None)  (aw_watcher_window.main:75)
2021-10-07 22:30:43 [ERROR]: Exception thrown while trying to get active window: (-2147217385, 'OLE error 0x80041017', None, None)  (aw_watcher_window.main:75)
2021-10-07 23:33:29 [ERROR]: Exception thrown while trying to get active window: (-2147217385, 'OLE error 0x80041017', None, None)  (aw_watcher_window.main:75)
```
There's more identical OLE errors

Maybe those relate to the "unknown" application.

**Comment by johan-bjareholt** [comment]
I think the screenshot is of the Linux build which does not have the same method of fetching the application name.
The behavior you have is expected on windows I believe.

**Comment by adil1214** [comment]
hi, im having this issue too on windows 11

**Comment by dassels** [comment]
Yes, it would be good to have the name show on Windows. I have the same issue with firefox.exe. When using different builds (like standard and developer edition), they all get reported as 'firefox.exe' so can't categorize them separately. Only work around so far was to rename the executable file so it the regex filters would be able to catch it. Not ideal as each time it gets updated I need to rename the file. If the window names were reported then one could categorize the different windows even though they are running under the same process name.
